documentation/usage.md

```diff
@@ -233,6 +233,95 @@ Suppliers.ORDER_BY(
   key.DESC(),
 )
 ```
+
+The following configs are used in the behavior of `DAYOFWEEK`, `DATETIME`, and `DATEDIFF`:
+
+5. `start_of_week` (default=`DayOfWeek.SUNDAY`): Determines which day is considered the first day of the week. This affects functions like 
+- `DAYOFWEEK` : An integer that returns the number of days since the start of the week. Start of week is relative to the `start_of_week` config.
```

Comment:
```suggestion
- `DAYOFWEEK` : A function that returns the number of days since the start of the week. Start of week is relative to the `start_of_week` config.
```